### DISCUSSION TITLE
最爱流萤

### DISCUSSION THREAD

**Discussion by yue0ying** [share121/inter-knot#2010] [其他]
![1$9RKO0D@J D1(J_N3I KHP](https://github.com/user-attachments/assets/8b54d145-5bfd-404d-a18a-142e98e4f0f2)
![I9 XRD}_VQ`2@1S2AW~I9FU](https://github.com/user-attachments/assets/ffe469dd-08d0-49c6-94f3-e40501ead536)

**Comment by YING-ADX** [comment]
绳匠崩铁好玩吗

**Comment by ghhccghk** [comment]
绳匠崩铁好玩吗

**Comment by nightrain2333** [comment]
QWQ<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Sacabambaspis114514** [comment]
![116323042-0](https://github.com/user-attachments/assets/96142be6-c080-475e-8a8d-023d1b4c2b76)
萤门！
